cms/envs/test.py

```diff
@@ -298,3 +298,8 @@
 DEFAULT_MOBILE_AVAILABLE = True
 
 PROCTORING_SETTINGS = {}
+
+############### ADG #####################
+
+MAILCHIMP_API_KEY = 'test'
+MAILCHIMP_LEARNERS_LIST_ID = 'test'
```

Comment:
Yes, tests are working with them, actually I have created a mocked MailChimp client, which does not need these Ids. They are added here just to avoid settings not found error.